content/docs/command-reference/plots/index.md

```diff
@@ -12,46 +12,37 @@ usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
 
 positional arguments:
   COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+    show         Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff         Show multiple versions of plot data by plotting them in a single image.
+    modify       Modify display properties of data-series plot outputs (has no effect on image-type plots).
 ```
 
-## Types of metrics
-
-DVC has two concepts for metrics, that represent different results of machine
-learning training or data processing:
-
-1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive rate_,
-   etc.
-2. `dvc plots` can be used to visualize **data series** such as AUC curves, loss
-   functions, confusion matrices, etc.
-
 ## Description
 
-DVC provides a set of commands to visualize certain metrics of machine learning
-experiments as plots. Usual plot examples are AUC curves, loss functions,
-confusion matrices, among others.
+DVC provides a set of commands to visualize data produced by machine learning
+experiments. Usual plot examples are AUC curves, loss functions, confusion
+matrices, among others.
 
-This type of metrics files are created by users, or generated by user data
-processing code, and can be defined in `dvc.yaml` (`plots` field) for tracking
-(optional).
+This type of data is created by users, or generated by user data processing
+code.
 
-DVC can work with two types of plots files:
+## Types of plots
+
+`dvc plots` is able to visualize two types of data:
 
 1. Data series files, which can be JSON, YAML, CSV or TSV.
 2. Image files in JPEG, GIF, or PNG format.
 
-DVC generates plots as static HTML webpages that can be open with a web browser.
-They can also be saved as SVG or PNG image files from the browser.
+DVC generates visualizations as static HTML webpages that can be open with a web
+browser. They can also be saved as SVG or PNG image files from the browser.
 
 Data-series plots utilize [Vega-Lite](https://vega.github.io/vega-lite/) for
-rendering (declarative JSON grammar for defining graphics). Image-type plots are
-rendered using `<img>` tags directly.
+rendering (declarative JSON grammar for defining graphics). Images are rendered
+using `<img>` tags directly.
 
 ## Supported file formats
```

Comment:
Hmmm... I see we already have all [this structure](https://dvc.org/doc/command-reference/plots#supported-file-formats) in. I'll make an issue about flattening this/ extracting to some guide ⏳  For now most of these should probably be H3s (contained under the **Description**)?

Reply:
> I'll make an issue about flattening this/ extracting to some guide 

A closely-related epic already exists so I just updated that one: https://github.com/iterative/dvc.org/issues/2572

Reply:
> Hmmm... I see we already have all [this structure](https://dvc.org/doc/command-reference/plots#supported-file-formats) in. I'll make an issue about flattening this/ extracting to some guide ⏳ For now most of these should probably be H3s (contained under the Description)?

You mean `Definig a plot too`?